test/unit/acknowledgement/process-invalid.test.js

```diff
@@ -0,0 +1,242 @@
+const mockCommit = jest.fn()
+const mockRollback = jest.fn()
+const mockTransaction = {
+  commit: mockCommit,
+  rollback: mockRollback
+}
+
+jest.mock('../../../app/data', () => {
+  return {
+    sequelize:
+       {
+         transaction: jest.fn().mockImplementation(() => {
+           return { ...mockTransaction }
+         })
+       }
+  }
+})
+
+jest.mock('../../../app/config')
+const mockConfig = require('../../../app/config')
+
+jest.mock('../../../app/event')
+const mockEvent = require('../../../app/event')
+
+jest.mock('../../../app/acknowledgement/get-hold-category-name')
+const getHoldCategoryName = require('../../../app/acknowledgement/get-hold-category-name')
+
+jest.mock('../../../app/holds/get-hold-category-id')
+const getHoldCategoryId = require('../../../app/holds/get-hold-category-id')
+
+jest.mock('../../../app/reschedule/index')
+const holdAndReschedule = require('../../../app/reschedule/index')
+
+jest.mock('../../../app/reset/index')
+const { resetPaymentRequestById } = require('../../../app/reset/index')
```

Comment:
Don't need trailing `/index` on these imports -- importing a module by nature, goes to the index file and imports it